Gemfile

```diff
@@ -7,6 +7,7 @@ gem "bootsnap", ">= 1.4.4", require: false
 gem "bootstrap-sass", "3.4.1"
 gem "config"
 gem "faker", "2.1.2"
+gem "figaro"
```

Comment:
gem này có tác dụng gì

Reply:
Gem firago giúp cấu hình các biến môi trường dễ dàng và an toàn hơn. Firago sẽ hỗ trợ đọc file config/application.yml và thiết lập các biến môi trường trước khi ứng dụng chạy.